docs/advanced/fields.rst

```diff
@@ -47,23 +47,15 @@ This approach has 2 main advantages:
 However, writing and maintaining ``to_item()`` method can get tedious,
 especially if there is a lot of properties.
 
-web_poet.fields
----------------
-
+@field decorator
+----------------
 To aid writing Page Objects in this style, ``web-poet`` provides
-a few utilities:
-
-* :func:`@web_poet.field <web_poet.fields.field>` decorator,
-* :func:`web_poet.item_from_fields <web_poet.fields.item_from_fields>`
-  and :func:`web_poet.item_from_fields_sync <web_poet.fields.item_from_fields_sync>`
```

Comment:
It seems we are removing `item_from_fields_sync` from the documentation here. If so, should we remove the function altogether? Otherwise, maybe we should cover it somewhere in the documentation, even if briefly.

Also, should `item_from_fields` become a private function?

Reply:
They are currently covered in API docs for web_poet.fields. 
No strong opinon on keeping them public vs private; I wonder if they can be useful for implementing custom to_item methods.